src/containers/twitter-embed/twitter.js

```diff
@@ -43,6 +45,6 @@ export default function Twitter() {
       </Suspense>
     );
   } else {
-    return null;
+    return <FailedLoading err={CUSTOM_ERR_MSG} />;
```

Comment:
This seems like a redundant implementation of the error message above, we can replace the above one with the `FailedLoading` component. 
Also, we should still show the Twitter header of the section to show which component didn't load instead of saying something didn't load.